src/main/java/study/StringCalculator.java

```diff
@@ -0,0 +1,44 @@
+package study;
+
+public class StringCalculator {
+
+    public double calculate(String input) throws Exception{
+        if(isBlank(input)) throw new IllegalArgumentException();
+
+        String[] arr = splitByWhiteSpace(input);
+        double calculatedValue = calculateByOperator(arr);
+
+        return calculatedValue;
+    }
+
+    boolean isBlank(String str) {
+        return str == null || str.trim().length() == 0;
+    }
+
+    String[] splitByWhiteSpace(String input) {
+        return input.split(" ");
+    }
+
+    double calculateByOperator(String[] arr) throws Exception {
+        double result = 0;
+
+        for(int i = 1; i < arr.length - 1; i+=2) {
```

Comment:
```suggestion
        for(int i = 1; i < arr.length - 1; i += 2) {
```
for 문 코딩 스타일이 지켜지면 좋을 것 같습니다!